libs/@hashintel/type-editor/src/entity-type-editor/link-list-card.tsx

```diff
@@ -140,6 +143,12 @@ const LinkTypeRow = ({
     editModalPopupState.close();
   };
 
+  const editDisabledReason = !canEditResource(linkNamespace)
+    ? "Can't edit link types that belong to other users or organizations you aren't a member of"
```

Comment:
This error message is specific to HASH and we might want to have multiple reasons for denying edit access in future. I would return the reason from the function, as `validateTitle` does.

Reply:
Done in 12c5824633ed9506136aae6ab04ff85ea6333c8a